scripts/sync-dashboards

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env python3
+
+import os
+import sys
+import argparse
+from urllib.request import urlretrieve
+
+# sync the dashboards from https://opendev.org/openstack/sunbeam-charms/src/branch/main/charms/openstack-exporter-k8s/src/grafana_dashboards
+BASE_URL = "https://opendev.org/openstack/sunbeam-charms/raw/branch/main/charms/openstack-exporter-k8s/src/grafana_dashboards/"
+BASE_LOCAL_PATH = "./src/grafana_dashboards/"
+DASHBOARDS = [
+    "hypervisor-openstack-exporter-dashboard.json",
+    "openstack-exporter.json",
+    "overview-openstack-exporter-dashboard.json",
+]
+
+
+def sync_dashboard(url: str, local_path: str, check_mode: bool) -> bool:
+    """Sync dashboard from url.
+
+    If check_mode is True, don't actually update the file.
+    Return True if the dashboard changed (or would be changed).
+    """
+    new_dashboard_path, _ = urlretrieve(url)
+
+    changed = False
+    if os.path.exists(local_path):
+        with open(new_dashboard_path) as f1, open(local_path) as f2:
+            changed = f1.read() != f2.read()
+    else:
+        changed = True
+
+    if changed and not check_mode:
```

Comment:
I would probably refactor the "sync" into a small functional call to avoid overloading a single function with two responsibility, but non-blocking since it's a small and simple script

Reply:
👌🏼